x/slashing/keeper/unjail.go

```diff
@@ -20,14 +19,6 @@ func (k Keeper) Unjail(ctx sdk.Context, validatorAddr sdk.ValAddress) error {
 		return types.ErrMissingSelfDelegation
 	}
 
-	tokens := validator.TokensFromShares(selfDel.GetShares()).TruncateInt()
```

Comment:
nit: I believe there are a few remaining references to MinSelfDelegation in `x/slashing/simulation/operations.go`

Reply:
good catch! It's possible it still passes since GetMinSelfDelegation just returns 0